descriptors/ModuleDescriptor-template.json

```diff
@@ -46,13 +66,25 @@
       "displayName": "Consortia API module - all permissions",
       "description": "All permissions for consortia module",
       "subPermissions": [
-        "consortia.tenants.collection.get"
+        "consortia.tenants.collection.get",
+        "consortia.user-tenants.collection.get",
+        "consortia.user-tenant.get"
       ]
     },
     {
       "permissionName": "consortia.tenants.collection.get",
       "displayName": "get tenants",
       "description": "Get list of tenants"
+    },
+    {
+      "permissionName": "consortia.user-tenants.collection.get",
+      "displayName": "get user tenants",
+      "description": "Get user tenants"
+    },
+    {
+      "permissionName": "consortia.user-tenant.get",
```

Comment:
Name should be: consortia.user-tenants.item.get

Reply:
Ok, I will fix